sunpy/lightcurve/sources/rhessi.py

```diff
@@ -51,7 +51,7 @@ class RHESSISummaryLightCurve(LightCurve):
     * Mission Paper `<http://link.springer.com/article/10.1023%2FA%3A1022428818870>`_
     """
 
-    def peek(self, title="RHESSI Observing Summary Count Rate", **kwargs):
+    def peek(self):
```

Comment:
what's wrong with this default title for rhessi? Is it coming from the data? should the title description in the docstring also be removed?